tests/test_citrine.py

```diff
@@ -22,14 +21,37 @@ def refresh_token(expiration: datetime = None) -> dict:
 
 
 def test_citrine_creation():
-
     with requests_mock.Mocker() as m:
         m.post('https://citrine-testing.fake/api/v1/tokens/refresh', json=token_refresh_response)
         m.get('https://citrine-testing.fake/api/v1/utils/runtime-config', json=dict())
 
         assert '1234' == Citrine(api_key='1234', host='citrine-testing.fake').session.refresh_token
 
 
+def test_citrine_signature():
```

Comment:
This should probably include a test that we can provide `api_key` as a positional argument without raising a `DeprecationWarning`.